old_timesheet_migration/2017_timesheet_migration.py

```diff
@@ -0,0 +1,253 @@
+import os
+import glob
+import pandas as pd
+from datetime import datetime, timedelta
+from config import settings
+from models import Activity, Client, Member, Project, TimeEntry
+
+
+def clean_timesheet(df, year):
+    '''Clean the timesheet of the year.'''
+
+    header = df.iloc[3]
+    for i, head in enumerate(header):
+        if head == 'Soma':
+            last_index = i - 1
+            break
+    dates = header[5:last_index].apply(lambda x: x[:5].replace('/', '-') + '-' + year)
+    header = header[:5].append(dates)
+    df = df[4:][df.columns[:last_index]]
+    df = df.rename(columns=header)
+    df = df.drop(['Curso'], axis=1)
+    df = df[df['Integrante'].notnull()]
+    df.update(df.fillna(0))
+    return df
+
+
+def check_row(project_name, activity_name, client_name):
+    '''Check if is a valid row based in project, activity and client.'''
+
+    if project_name in ('Feriado', 'Falta justificada'):
```

Comment:
Ok, I don't know if I really understand, because the code was made for old timesheets and this is the only two cases that have to be filtered. Can you suggest how can I encapsulate this?